tests/conftest.py

```diff
@@ -41,7 +39,7 @@ def params_Kaminski2001_fig5_solid():
         "gbm_mobility": 0,
         "gbs_threshold": 0,
         "nucleation_efficiency": 5,
-        "minerals": ("olivine"),
```

Comment:
I think I wanted for these to be a tuple so the type is predictable, but for that it should be `("olivine",)` with the comma, which I forgot about.